### PR TITLE
Dev/fix ubt race no async

### PR DESCRIPTION
**Pull request by DecoyRS**

### CODE REVIEW

File: src/dotnet/RiderPlugin.UnrealLink/PluginInstaller/UnrealPluginInstaller.cs [hunk 1]
```diff
@@ -503,109 +501,138 @@ private bool GenerateProjectFilesCmd(FileSystemPath UprojectFile, FileSystemPath
...
-                    myLogger.Info($"[UnrealLink]: Regenerating project files: {commandLine}");
-                    ErrorLevelException.ThrowIfNonZero(InvokeChildProcess.InvokeChildProcessIntoLogger(
-                        BatchUtils.GetPathToCmd(),
-                        commandLine,
-                        LoggingLevel.INFO,
-                        TimeSpan.FromMinutes(1),
-                        InvokeChildProcess.TreatStderr.AsOutput,
-                        pathToUnrealVersionSelector.Directory
-                    ));
+                    Console.Out.WriteLine("Hello~!");
```
Comment (real-mikhail) [inline src/dotnet/RiderPlugin.UnrealLink/PluginInstaller/UnrealPluginInstaller.cs:584]: Hi! 😃

**Review by real-mikhail — CHANGES_REQUESTED** [review]
Added one comment. Also is it me or lines from 385 to 388 contain some nonsense?

File: src/dotnet/RiderPlugin.UnrealLink/PluginInstaller/UnrealPluginInstaller.cs (hunk 1 above)
Comment (DecoyRS) [reply src/dotnet/RiderPlugin.UnrealLink/PluginInstaller/UnrealPluginInstaller.cs:584]: rofl

**Comment by DecoyRS** [comment]
@real-mikhail jeez, I wonder how long this comment was hiding there. Good catch! I'll nuke it

**Review by real-mikhail — APPROVED** [review]

**Pull request merged**